src/hooks/chatSidebar.ts

```diff
@@ -83,3 +89,24 @@ export const useChatSidebarInfo = () => {
     venueChatTabTitle: `${chatTitle} Chat`,
   };
 };
+
+export const useChatSidebarPinned = () => {
+  const dispatch = useDispatch();
+  const { width } = useWindowSize();
+  const venue = useSelector(currentVenueSelectorData);
```

Comment:
We probably don't want to be using this 'raw selector' here like this.

Since `src/pages/VenuePage/TemplateWrapper.tsx` already has the `venue` in-scope, and it renders `src/components/organisms/ChatSidebar/ChatSidebar.tsx`, which then calls this `useChatSidebarPinned` hook; perhaps we could pass the `venue` as a prop to both `ChatSidebar` and then `useChatSidebarPinned`?

Reply:
fixed. Looks better